sklearn/covariance/robust_covariance.py

```diff
@@ -659,6 +659,13 @@ def correct_covariance(self, data):
             The data set must be the one which was used to compute
             the raw estimates.
 
+        References
+        ----------
+
+        .. [Rouseeuw1999] `A Fast Algorithm for the Minimum Covariance
```

Comment:
I think it's auto generating the number as the same tag has been used in the file several times.